scipy/linalg/tests/test_basic.py

```diff
@@ -1441,6 +1442,12 @@ def test_atol_rtol(self):
         assert_allclose(np.linalg.norm(adiff1), 4.233, rtol=0.01)
         assert_allclose(np.linalg.norm(adiff2), 4.233, rtol=0.01)
 
+    @pytest.mark.parametrize(["cond", "rcond"],
+                             [(1, 1), (1, _NoValue), (_NoValue, 1)])
```

Comment:
Can you parametrize them separately please, and use `[1, None, _NoValue]` for both?